File: sort_colors/sol.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        lst = [0,0,0]
        for n in nums:
            lst[n] +=1
        print(lst)
        j = 0
        i = 0
        for n in lst:
            nums[i:i+n] = [j]*n
            print(nums)
            i += n
            j+=1

File: sort_colors/sol.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int arr[3] ={0};
        for (int a: nums)
            arr[a]++;
        
        int i = 0;
        int k = 0;
        int j = 0;  
        while(i < 3)
        {      
            while(j < arr[i])
            {
                nums[k]=i;
                j++;
                k++;
            }
            i++;
            j =0;
        }
    }
};